src/main/java/com/aws/greengrass/mqtt/bridge/clients/LocalMqtt5Client.java

```diff
@@ -482,42 +474,70 @@ private Mqtt5Client createCrtClient() throws MessageClientException {
         }
 
         try {
-            Mqtt5ClientOptions.Mqtt5ClientOptionsBuilder builder =
-                    new Mqtt5ClientOptions.Mqtt5ClientOptionsBuilder(brokerUri.getHost(), port)
-                            .withLifecycleEvents(connectionEventCallback)
-                            .withPublishEvents(publishEventsCallback)
-                            .withSessionBehavior(Mqtt5ClientOptions.ClientSessionBehavior.REJOIN_POST_SUCCESS)
-                            .withOfflineQueueBehavior(
-                                    Mqtt5ClientOptions.ClientOfflineQueueBehavior.FAIL_ALL_ON_DISCONNECT)
-                            .withConnectOptions(new ConnectPacket.ConnectPacketBuilder()
-                                    .withRequestProblemInformation(true)
-                                    .withClientId(clientId).build())
-                            // TODO configurable?
-                            .withMaxReconnectDelayMs(Duration.ofSeconds(MAX_RECONNECT_DELAY_SECONDS).toMillis())
-                            .withMinReconnectDelayMs(Duration.ofSeconds(MIN_RECONNECT_DELAY_SECONDS).toMillis());
+            Mqtt5ClientOptions.Mqtt5ClientOptionsBuilder builder
+                    = new Mqtt5ClientOptions.Mqtt5ClientOptionsBuilder(brokerUri.getHost(), port)
+                    .withLifecycleEvents(connectionEventCallback)
+                    .withPublishEvents(publishEventsCallback)
+                    .withSessionBehavior(Mqtt5ClientOptions.ClientSessionBehavior.REJOIN_POST_SUCCESS)
+                    .withOfflineQueueBehavior(
+                            Mqtt5ClientOptions.ClientOfflineQueueBehavior.FAIL_ALL_ON_DISCONNECT)
+                    .withConnectOptions(new ConnectPacket.ConnectPacketBuilder()
+                            .withRequestProblemInformation(true)
+                            .withClientId(clientId).build())
+                    // TODO configurable?
+                    .withMaxReconnectDelayMs(Duration.ofSeconds(MAX_RECONNECT_DELAY_SECONDS).toMillis())
+                    .withMinReconnectDelayMs(Duration.ofSeconds(MIN_RECONNECT_DELAY_SECONDS).toMillis());
+
+            TlsContext tlsContext = null;
+            TlsContextOptions tlsContextOptions = null;
 
             if (isSSL) {
-                mqttClientKeyStore.listenToCAUpdates(onKeyStoreUpdate);
-                this.tlsContextOptions = TlsContextOptions.createWithMtlsJavaKeystore(mqttClientKeyStore.getKeyStore(),
-                        KEY_ALIAS, new String(DEFAULT_KEYSTORE_PASSWORD));
-                this.tlsContext = new ClientTlsContext(tlsContextOptions);
+                tlsContextOptions = TlsContextOptions.createWithMtlsJavaKeystore(
```

Comment:
is this code path tested at all yet? We don't have #116 merged yet

Reply:
tested via the UAT, but nowhere else yet until #116

Reply:
can we get #116 in then to get this validated?

Reply:
yeah i put that on the backburner for this pr, last i worked on it was getting some crt errors, but will take another stab at it after this.  i won't merge this until the uat i kicked off passes